keras/backend/tensorflow_backend.py

```diff
@@ -725,10 +725,7 @@ def eye(size, dtype=None, name=None):
     ```
 
     """
-    if dtype is None:
-        dtype = floatx()
-    tf_dtype = tf.as_dtype(dtype)
-    return variable(tf.eye(size, dtype=tf_dtype), dtype, name)
+    return variable(np.eye(size), dtype, name)
```

Comment:
why np.eye?

Reply:
Oops, accidental revert to an earlier version. Thanks for the catch.